sdk/go/auto/cmd.go

```diff
@@ -38,9 +224,14 @@ func runPulumiCommandSync(
 	// all commands should be run in non-interactive mode.
 	// this causes commands to fail rather than prompting for input (and thus hanging indefinitely)
 	args = withNonInteractiveArg(args)
-	cmd := exec.CommandContext(ctx, "pulumi", args...)
+	cmd := exec.CommandContext(ctx, p.command, args...) // #nosec G204
```

Comment:
This can stay as it is, but it does make me wonder, since we're appending to the `$PATH` below anyway, could we just keep executing the `pulumi` binary and keep relying on the `$PATH`? I'm not even sure it matters, just noticed it because the `// #nosec` comment stood out.  I think that's also fine to keep, but I figured I'd write down my musings :)

Reply:
The binary might not be called "pulumi"

Reply:
Hmmm we always assume that the binary is `$ROOT/bin/pulumi`

Reply:
Ah yeh then I think Thomas is right